src/v3/src/util/formUtils.ts

```diff
@@ -262,3 +263,34 @@ export const getIdpButtonElements = (
 
   return [...pivButtonElement, ...idpButtonElements];
 };
+
+export const getBiometricsErrorMessageElement = (messageKey: string | undefined, displayName = 'Custom App'): WidgetMessage => {
```

Comment:
After looking more at this, I think we can just get rid of the default value here, it _should_ be set when we use `getDisplayName`, I think I see why you added the default value (since displayName is undefinable) but I think if we expect a value from the server, we shouldn't substitute it with something unexpected if for some reason the server breaks and stops sending us the expected value. We should show the error so it can be realized and fixed.